.github/workflows/tests.yml

```diff
@@ -178,9 +178,6 @@ jobs:
         run: |
           brew audit --skip-style --except=version --tap=homebrew/cask
 
-      - name: Generate formula API
-        run: brew generate-formula-api --dry-run
```

Comment:
Ah, no, I see. This step runs on macOS, but the other one runs on Linux. Do we really need both?

Reply:
@carlocab Yes. This was added by when we had formulae generation failing on one platform but not another.